src/index.ts

```diff
@@ -76,7 +76,7 @@ import versionInfo from './versionInfo'
 const dom = window ? window.document : null // Idea that UI.dom can be adapted in non-browser environments
 
 if (typeof window !== 'undefined') {
-  ;(<any>window).UI = {
+  ; (<any>window).UI = {
```

Comment:
Is this because of ESLint + Standard? Haven't noticed this before, so little strange to see these small variations...